### PR TITLE
Fix sourcecode install guide for new version of kubeedge

### LINKED ISSUE #1522: Can not find cloudcore in $GOPATH/src/github.com/kubeedge/kubeedge/cloud
**What happened**:kubeedge_install_source.md ask me to:
```
cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud
# run cloudcore controller
# verify the configurations before running cloud(cloudcore)
mkdir ~/cmd/
cp cloudcore ~/cmd/
```
But I can't find cloudcore.

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1522

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/getting-started.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ Learn about KubeEdge and the KubeEdge Mission [here](../modules/kubeedge.md)
 
 ### First Steps  
 To get the most out of KubeEdge, start by reviewing a few introductory topics:  
-- [Setup](../setup/setup.md) - Install KubeEdge  
+- Quick Start - [Install KubeEdge with keadm](../setup/kubeedge_install_keadm.md) - Install KubeEdge
+- [Start developing Kubeedge](../setup/develop_kubeedge.md)
```
Comment (kevin-wangzefeng) [inline docs/getting-started/getting-started.md]:
```suggestion
- Quick Start - [Install KubeEdge with keadm](../setup/kubeedge_install_keadm.md)
- [Start developing KubeEdge](../setup/develop_kubeedge.md)
```

File: docs/setup/deploy-edge-node.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
-cat>./node.json<<EOF
-{
-  "kind": "Node",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": $nodename,
-    "labels": {
-      "name": "$nodename",
-      "node-role.kubernetes.io/edge": ""
++ Copy the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the name of the edge node
```
Comment (kevin-wangzefeng) [inline docs/setup/deploy-edge-node.md]:
```suggestion
+ Copy `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` to your working directory and change `metadata.name` to the name of edge node
```

**Comment by kevin-wangzefeng** [comment]
Minor comments

**Comment by fisherxu** [comment]
> Minor comments

Done

**Pull request merged**